### ISSUE TITLE
Citation link for Relion 

### ISSUE THREAD

**Issue by mcianfrocco** [leschzinerlab/COSMIC-CryoEM-Gateway#21]
Current citation link isn't working for SHW Scheres for a given task. Needs to be updated to include: 

http://www.sciencedirect.com/science/article/pii/S1047847712002481

**Comment by mcianfrocco** [comment]
Click out link has this listing: 

gateway/relion_3d_class_comet!input.action#REFERENCE

**Comment by kenneth59715** [comment]
Is there still a bad link somewhere?

**Comment by mcianfrocco** [comment]
Yes, these links still don't go anyway even though I've tried to update them. This link appears when you are inputting parameters for a given job under the "Set parameters" tab.

**Comment by kenneth59715** [comment]
The #REFERENCE link is also broken for NSG and phylo.org tools, so I think it's
in the code to generate the tool .jsp files, rather than a misconfiguration in our tool
xml files.  I think pise2JSP.ftl could be modified to include the correct reference
text...

**Comment by kenneth59715** [comment]
Okay, I modified pise2JSP.ftl to put in a link from the tool xml reference attribute.
relion_2d_class_comet.xml has this mod.  Probably the other tools need the
reference attribute changed to a URI, then this should show up in the Set Parameters
link.

**Comment by kenneth59715** [comment]
Michael, could you check the link under Set Parameters for 2d, then if it looks okay,
send me URIs for the other tools, and I can modify the tool xml files.

**Comment by mcianfrocco** [comment]
That works - thanks! 

Other tools: 
Relion 3D classification (3.0-beta)   - Perform 3D classification of single particles with 3D model using Relion
Relion 3D initial model calculation (3.0-beta)   - Calculate initial 3D models using Relion
Relion 3D refinement (3.0-beta)   - I

**Comment by kenneth59715** [comment]
Okay, I put that reference link into the other tools as well.

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
